docs/tutorials/quickstart.md

```diff
@@ -140,7 +151,7 @@ For more on automatic differentiation in JAX, check out {ref}`automatic-differen
 
 Another useful transformation is {func}`~jax.vmap`, the vectorizing map.
 It has the familiar semantics of mapping a function along array axes, but instead of keeping the loop on the outside, it pushes the loop down into a function’s primitive operations for better performance.
-When composed with {func}`~jax.jit`, it can be just as fast as adding the batch dimensions manually.
+When composed with {func}`~jax.jit`, it can be just as fast as rewriting your function to handle dimensions manually.
```

Comment:
I kinda thought mentioning the batch dimension was more clear?
```suggestion
When composed with {func}`~jax.jit`, it can be just as performant as manually rewriting your function to operate over an extra batch dimension.
```

Reply:
Done